src/sentinel.c

```diff
@@ -1739,7 +1739,13 @@ char *sentinelHandleConfiguration(char **argv, int argc) {
         if ((slave = createSentinelRedisInstance(NULL,SRI_SLAVE,argv[2],
                     atoi(argv[3]), ri->quorum, ri)) == NULL)
         {
-            return "Wrong hostname or port for replica.";
+            switch(errno) {
```

Comment:
@hwware Please consider consolidating these two switch statements with the already existing one? Also, I think it's a good idea to have a `default:` with a generic error both as safety and to silence compiler warnings around unhandled cases.

Reply:
thank you @yossigo , I missed one part about this, for sentinel we are using runid as the name, so sentinelCheckCreateInstanceErrors can have three cases. Please review my new changes if you think it make sense. Thank you!